### ISSUE TITLE
[Question]Poor detection performance of own cerebral aneurysm files

### ISSUE THREAD

**Issue by hty211** [MIC-DKFZ/nnDetection#336]
### :question: Question
All-positive data, but the validation set consistently failed to identify positive samples during training. After training, the prediction results are extremely poor, manifested as multiple large bounding boxes occupying most of the brain, showing a huge gap with the ground truth masks. It is far from meeting the results of the ADAM Challenge.

**Comment by andres2631996** [comment]
Sorry for the delay in the answer. Have you checked that your instance segmentation labels in _raw_splitted/labelsTr_ make sense and are aligned with the actual aneurysms in the respective images? You can check this in any medical image visualizer such as MITK, ITK Snap, 3D Slicer, etc. 

Have you seen any warnings in your preprocess.log and train.log files? Which configuration are you using? The default RetinaUNet?

How does your loss function evolve during training? Does it reduce with the epochs? Do your validation metrics during training also improve with the epochs?

This looks to me like a problem with the labels, but I would need more information about your problem in order to guide you better. The sample selection for the dataloader is executed here: [https://github.com/MIC-DKFZ/nnDetection/blob/main/nndet/io/datamodule/bg_loader.py#L154](url)

If the issue is really in the dataloader, you can change the dataloader in the configuration file from [https://github.com/MIC-DKFZ/nnDetection/blob/main/nndet/conf/train/v001.yaml#L16](url) from "DataLoader{}DOffset" to "DataLoader{}DBalanced"
